### ISSUE TITLE
This project should be marked as deprecated / no longer maintained.

### ISSUE THREAD

**Issue by thibka** [brocessing/kirby-webpack#67]
Hi,

It seems the project is no longer maintained :( 
You should mention it in the readme as it won't work with Kirby 3.

**Comment by pqml** [comment]
hi, we don't have a lot of time to maintain the repo but the v3 branch should work with kirby v3
https://github.com/brocessing/kirby-webpack/tree/kirby-3

**Comment by chrtz** [comment]
@pqml The kirby-3 branch wouldn't work out of the box anymore and is pretty much outdated technically seen, too.
My Pull-Requests (I updated some dependencies, etc.) which I provided 8 months ago weren't merged till now.
And even these are outdated by now I guess.

I was thinking to prepare a new package of Kirby Webpack from scratch, but same as the @brocessing team, I haven't any spare time to do so.

I still appreciate the brocessings work, but this repo is more or less dead by now.
